packages/core/src/common/keys.ts

```diff
@@ -114,6 +153,19 @@ export class KeyCode {
         throw new Error(`Cannot get key code from the keyboard event: ${event}.`);
     }
 
+    public static equals(keycode1: KeyCode, keycode2: KeyCode): boolean {
+
+        if (keycode1.key && keycode2.key) {
+            return keycode1.alt === keycode2.alt &&
+                keycode1.ctrl === keycode2.ctrl &&
+                keycode1.meta === keycode2.meta &&
+                keycode1.shift === keycode2.shift && keycode1.key.keyCode === keycode2.key.keyCode &&
+                keycode1.key.code === keycode2.key.code;
```

Comment:
We should compare keycode1.key with keycode2.key directly you could use JSON.stringify() for that since the object have the same properties order and are simple.

Reply:
So just to be clear I should use JSON.stringify(object1) === JSON.stringify(object2)? That makes it much cleaner I agree.

Reply:
Yes it's just that if you add an element in object1 you won't break your compare here.